src/main/kotlin/ink/abb/pogo/scraper/tasks/CatchOneNearbyPokemon.kt

```diff
@@ -53,7 +53,8 @@ class CatchOneNearbyPokemon : Task {
 
                     if (result.status == CatchPokemonResponse.CatchStatus.CATCH_SUCCESS) {
                         ctx.pokemonStats.first.andIncrement
-                        var message = "Caught a ${catchablePokemon.pokemonId} with CP ${encounterResult.wildPokemon.pokemonData.cp} using $ball"
+                        val iv = (encounterResult.wildPokemon.pokemonData.individualAttack + encounterResult.wildPokemon.pokemonData.individualDefense + encounterResult.wildPokemon.pokemonData.individualStamina) * 100 / 45
```

Comment:
Please use encounterResult.wildPokemon.ivPercentage (or I am mistaken and that is in a different class, in which case extend the extension function in util/pokemon/Pokemon.kt)